network/portrange.go

```diff
@@ -196,3 +196,28 @@ func parsePortRange(portRange string) (PortRange, error) {
 	}
 	return result, nil
 }
+
+// CombinePortRanges groups together all port ranges according to
```

Comment:
please call out in flashing capital letters that this only works on non-overlapping sets